### ISSUE TITLE
Triangulation with holes / nested contours

### ISSUE THREAD

**Issue by Evil-Spirit** [NoteCAD/NoteCAD#27]
Vote with emoji for implementing it.

**Comment by manaspatnaik** [comment]
What's the status of this feature are you working on it? IF not I can make an effort .

I have done the same useing .NetTopology Suite off course on another cad package.

**Comment by Evil-Spirit** [comment]
Yes, this is coming soon. I was going to use triangle.net, but if .nettopology is better then I will use it

**Comment by Evil-Spirit** [comment]
done by copilot
